mls-rs/src/group/proposal_filter/filtering_common.rs

```diff
@@ -482,17 +482,20 @@ fn ensure_exactly_one_external_init(proposals: &ProposalBundle) -> Result<(), Ml
         .ok_or(MlsError::ExternalCommitMustHaveExactlyOneExternalInit)
 }
 
+/// Non-default proposal types are by default allowed. Custom MlsRules may disallow
+/// specific custom proposals in external commits
 fn ensure_proposals_in_external_commit_are_allowed(
     proposals: &ProposalBundle,
 ) -> Result<(), MlsError> {
-    let unsupported_type = proposals.proposal_types().find(|ty| {
-        ![
-            ProposalType::EXTERNAL_INIT,
-            ProposalType::REMOVE,
-            ProposalType::PSK,
-        ]
-        .contains(ty)
-    });
+    let supported_default_types = [
+        ProposalType::EXTERNAL_INIT,
+        ProposalType::REMOVE,
+        ProposalType::PSK,
```

Comment:
I was trying to figure out a different time where in the RFC this is defined?

Reply:
Doesn't seem to be, which is less than ideal :(